test/js/main.js

```diff
@@ -0,0 +1,191 @@
+//require("@babel/register");
+const {Builder} = require('selenium-webdriver');
+const remote = require('selenium-webdriver/remote');
+const config = require('./config.json');
+const Mocha = require('mocha');
+import {createBrowserStackLocal,stopBrowserstackLocal} from './utils/browserstack'
```

Comment:
[Minor] This and below imports have no semicolons at the end of lines, there's no ESLint rule set for this?..

Reply:
I'm sure there is a ruleset, but not on our config it seems